src/nimblepkg/init.nim

```diff
@@ -14,6 +14,8 @@ type
     pkgNimDep: string
     pkgType: string
 
+template p(s: string{lit}): string = static(s.replace("\n", "\p"))
```

Comment:
Hold up, Nim doesn't use platform-specific newlines in multi-line string literals? That seems like a bug to me, and indeed the reason Nimble used \n here is because in the past it had the same meaning as `\p`.

Can you create an issue on the Nim repo and ask about this?

Reply:
Nim always uses LF for ``\n`` and that is fine since every Windows tool in widespread use understands LF.